test/SILOptimizer/lslocation_expansion.sil

```diff
@@ -138,87 +138,9 @@ sil @store_after_store_struct : $@convention(thin) () -> () {
 // CHECK: #1 store
 // CHECK-NEXT: alloc_stack $S3
 // CHECK-NEXT: Projection Path [$*S3
-// CHECK-NEXT:   Field: var c: S2 of: $*S2
```

Comment:
The problem with just deleting this code is that this code is making sure that this file is actually testing lslocation's ability to expand structs at all.

Can you add a flag to force expanding and use that here. Then you can (perhaps with a second RUN line/FileCheck) check it with expanding disabled (giving you what you are looking for, proof that you are not expanding).

Reply:
We were discussing having multiple levels of expansions / a user-facing flag (maybe depending on the optimization / code-size level) - we will add that in a later date which would also cover this test case.

Reply:
How soon are you talking about doing that work?

I would really really not like to lose this code coverage. How about just adding an LLVM option at expand that just causes it to always return true. It is not user exposed and just for testing. Then copy/paste the FileCheck tests with a different prefix.

Reply:
Alright I added a frontend option and turned it on for those tests.